modules/cold-extraction/config.json

```diff
@@ -10,6 +10,7 @@
 	"SecondIndex": 5,
 	"ThirdAttr": "StudyDate",
 	"ThirdIndex": 3,
+	"LongAccession": true,
```

Comment:
default should be false.

Reply:
Modified the source code as per comments